docs/whatsnew/3.2.rst

```diff
@@ -106,6 +135,37 @@ conventions used in particular fields. For more details on the motivation behind
 these changes, see `PR #8394 <https://github.com/astropy/astropy/pull/8394>`_
 and the associated discussion.
 
+As an example, this shows the evolution of the ecliptic origin for the true and
+mean barycentric ecliptic frames over the course of a year:
+
+
+.. plot::
+    :include-source:
+    :context: reset
+
+    import numpy as np
+    import matplotlib.pyplot as plt
+
+    from astropy import units as u
+    from astropy.time import Time
+    from astropy.coordinates import BarycentricMeanEcliptic, BarycentricTrueEcliptic, ICRS
+
+    t = Time('J2018') + np.linspace(-1, 0, 1000)*u.year
+
+    bme_origin = BarycentricMeanEcliptic([0]*len(t)*u.deg, [0]*len(t)*u.deg, equinox=t)
```

Comment:
Not directly relevant here, but should we consider the fact that we do not just broadcast the coordinates with the times a bug?

Reply:
Good point.  I was surprised by that, and just accepted it, but if it surprised me maybe I shouldn't have!  Will make an issue.